data/json/itemgroups/food_service.json

```
@@ -976,5 +976,49 @@
       { "item": "stout", "prob": 10, "container-item": "keg" },
       { "item": "imperial_stout", "prob": 5, "container-item": "keg" }
     ]
+  },
+  {
+    "type": "item_group",
+    "id": "butcher_raw_meat",
+    "items": [
+      [ "meat", 8 ],
```

```suggestion
      [ "stomach_larget", 2 ],
      [ "stomach_boiled", 2 ],
      [ "meat", 8 ],
```

Should add raw and boiled large stomachs to account for green and gray tripe.